bumble/hci.py

```diff
@@ -5853,12 +6045,92 @@ def __str__(self) -> str:
         return (
             f'{color("SCO", "blue")}: '
             f'handle=0x{self.connection_handle:04x}, '
-            f'ps={self.packet_status}, rfu={self.rfu}, '
+            f'ps={self.packet_status}, '
             f'data_total_length={self.data_total_length}, '
             f'data={self.data.hex()}'
         )
 
 
+# -----------------------------------------------------------------------------
+class HCI_IsoDataPacket(HCI_Packet):
+    '''
+    See Bluetooth spec @ 5.4.5 HCI ISO Data Packets
+    '''
+
+    hci_packet_type = HCI_ISO_DATA_PACKET
+
+    @staticmethod
+    def from_bytes(packet: bytes) -> HCI_IsoDataPacket:
+        h = int.from_bytes(packet[0:2], 'little')
+        connection_handle = h & 0xFFF
+        pb_flag = (h >> 12) & 0x0F
+        ts_flag = (h >> 12) & 0x20
+        data_total_length = int.from_bytes(packet[2:4], 'little')
+        time_stamp = int.from_bytes(packet[4:8], 'little')
```

Comment:
The spec says that `If PB_Flag is 0b01 or 0b11, then Data_Total_Length may be zero`, so the following fields may not exist for all packets.

Reply:
Done